plugins/tech-insights-backend-module-jsonfc/src/service/JsonRulesEngineFactChecker.test.ts

```diff
@@ -122,7 +142,16 @@ const latestSchemasMock = jest.fn().mockImplementation(() => [
   },
 ]);
 const factsBetweenTimestampsByIdsMock = jest.fn();
-const latestFactsByIdsMock = jest.fn().mockImplementation(() => ({}));
+const latestFactsByIdsMock = jest.fn().mockImplementation(() =>
+  Promise.resolve({
+    ['test-factretriever']: {
+      id: 'test-factretriever',
+      facts: {
+        testnumberfact: 3,
+      },
+    },
+  }),
+);
```

Comment:
Would this do the same thing?

```suggestion
const latestFactsByIdsMock = jest.fn().mockResolvedValue({
  ['test-factretriever']: {
    id: 'test-factretriever',
    facts: {
      testnumberfact: 3,
    },
  },
});
```